courses/machine_learning/deepdive2/image_classification/solutions/3_tf_hub_transfer_learning.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<p>if you add a ";" after plt.imshow((img.numpy())), it won't show the &lt;matplotlib.....&gt;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>TIL, thanks!</p>